src/internal/m365/graph/errors.go

```diff
@@ -38,11 +38,15 @@ const (
 	nameAlreadyExists       errorCode = "nameAlreadyExists"
 	quotaExceeded           errorCode = "ErrorQuotaExceeded"
 	RequestResourceNotFound errorCode = "Request_ResourceNotFound"
-	resourceNotFound        errorCode = "ResourceNotFound"
-	resyncRequired          errorCode = "ResyncRequired" // alt: resyncRequired
-	syncFolderNotFound      errorCode = "ErrorSyncFolderNotFound"
-	syncStateInvalid        errorCode = "SyncStateInvalid"
-	syncStateNotFound       errorCode = "SyncStateNotFound"
+	// Returned when we try to get the inbox of a user that doesn't exist.
+	resourceNotFound errorCode = "ResourceNotFound"
+	// Some datacenters are returning this when we try to get the inbox of a user
+	// that doesn't exist.
+	invalidUser        errorCode = "ErrorInvalidUser"
+	resyncRequired     errorCode = "ResyncRequired" // alt: resyncRequired
```

Comment:
I know it's old code, but can you remove the `alt` comment here?  We should be lowering text for comparisons.